.github/scripts/release-ecr-tags.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Release Tags
+ *
+ * Creating release tag based on each release version for AWS ECR Public
+ *
+ */
+
+module.exports = ({ context }) => {
+  if (context.eventName === 'release') {
+    return getReleaseTag(context)
+  }
+  throw new Error('Release Violation: Could not determine the required release tags.')
+}
+
+function getReleaseTag(domain, app, context) {
+  const semver = context.payload.release.tag_name.replace('v', '')
+  if (semver.match(/^[0-9]+\.[0-9]+\.[0-9]+$/) === false) {
```

Comment:
1) match will always return something if able to match (For eg: `[ '1.1.1', index: 0, input: '1.1.1', groups: undefined ]` for `1.1.1`)
2) match will always return null if unable to match (For eg: null for `22`)

As it is impossible to get false for ALL THE SCENARIO, this line `Release Violation: Could not determine the required release tags` will never be executed

See match docs below
<img width="814" alt="Screenshot 2022-05-01 at 5 39 20 PM" src="https://user-images.githubusercontent.com/7101806/166140475-a0cf1396-1b37-406a-8f33-f87d9a67134a.png">

You maybe can write something like this
```suggestion
  if (semver.match(/^[0-9]+\.[0-9]+\.[0-9]+$/) === null) {
```

Reply:
Maybe can remove this line `context.payload.release.tag_name.replace('v', '')`
If not your code may allow other strings like `vv1.1.1vv` or `v1.v1.v1`

If you agree with me kindly change the same code you referred to in Jellyfish

Reply:
Thanks @jingyi2811 for bringing this up. The existing jellyfish code replaces only the first 'v' which means strings like `vv1.1.1vv` or `v1.v1.v1` will not pass the regex condition, which was why it will work for such scenarios.

But I agree that your approach will be more fool proof, where currently strings without the `v` such as `1.1.1` will pass the regex. Let me update it on both existing and new file.